python/CMakeLists.txt

```diff
@@ -2,14 +2,19 @@
 # This file is distributed under the MIT License. See LICENSE.md for details.
 #
 
-# Declares a python module as a target Named arguments: - TARGET_NAME: the name
-# of the CMake target to be created - MODULE_FILES: list of relative paths of
-# the python module sources - MODULE_GENERATED_FILES: list of relative paths of
-# the python module generated sources The module sources will be copied to the
-# build directory (in the lib/python folder) and installed in /lib/python.
+# ~~~
```

Comment:
What are these? I'd leave an empty comment line if you want to better isolate this part.

Reply:
That's a fence for `cmakeformat` (See [README](https://github.com/cheshirekow/cmake_format), search for 'fences:'). It avoids the whitespace being collapsed, leading to the previous state of the comments where the bullet point list was squashed

Reply:
Right right, I forgot.